UnicornEngine/video/source/Renderer.cpp

```diff
@@ -19,7 +19,7 @@ Renderer::Renderer(system::Manager& manager, system::Window* window) : m_isIniti
                                                                      , m_backgroundColor({{0.0f, 0.0f, 0.0f, 0.0f}})
                                                                      , m_depthTestEnabled(true)
 {
-    m_camera.SetPerspective(45.0f, static_cast<float>(window->GetSize().first) / window->GetSize().second, 0.1f, 100.0f);
+    m_camera.SetPerspective(45.0f, static_cast<float>(window->GetSize().first) / window->GetSize().second, 0.1f, 100000.0f);
```

Comment:
Move it to a member variable?

Reply:
Why? It can change and it's not Renderer.cpp 's variables.
I will split SetPerspective

Reply:
Those are fault values, would be a lot more useful to have those configurable from the more clear place in the code.